openedx/features/classrooms/models.py

```diff
@@ -0,0 +1,13 @@
+from nodebb.models import DiscussionCommunity
+from model_utils.models import TimeStampedModel
+
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class DiscussionCommunityMembership(TimeStampedModel):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    community = models.ForeignKey(DiscussionCommunity, on_delete=models.CASCADE)
```

Comment:
Shall we make the column name `community_url` as it is in `DiscussionCommunity`?

Reply:
That's a good suggestion. Implemented it now.

Reply:
Did we change name of the column to `community_url` after this conversation? @mabdullahz @abdullah998 community or discussion_community does make more sense here. Image accessing url from this FK `community.community_url` makes more sense or `community_url.community_url`? also in database it append _id with column name so make a comparison of `community_url_id` or `discussion_community_id`?

Reply:
Did we change name of the column to `community_url` after this conversation? @mabdullahz @abdullah998 community or discussion_community does make more sense here. Image accessing url from this FK `community.community_url` makes more sense or `community_url.community_url`? also in database it append _id with column name so make a comparison of `community_url_id` or `discussion_community_id`?

Reply:
@ahmed-faraz46 I get your point, I have now changed it back to community.